### PR TITLE
Brbartle/newregistrationscript

### PR DESCRIPTION
**Pull request by BradleyBartlett**
New registration script to go along with the new flow of registration.

### CODE REVIEW

**Comment by sarathys** [comment]
Note this isn’t always true, its prefix-ercs01, where prefix is Azs for 1-node:

+.\RegisterWithAzure.ps1 -CloudAdminCredential $CloudAdminCredential -AzureSubscriptionId $SubscriptionId -JeaComputername "Azs-ERC01"

when checking for versions use the strongly typed Version object.

E.g:

$fileVer = [Version] "1.0.0.0"

or

$ver2 = $null
$result = [Version]::TryParse("1.0.0.0", [ref]$ver2)
$ver2

File: Registration/RegisterWithAzure.ps1 [hunk 1]
```diff
@@ -5,224 +5,277 @@
...
+This example registers your AzureStack account with Azure, enables syndication and usage and gives a specific name to the resource group and registration resource. 
 
+.\RegisterWithAzure.ps1 -CloudAdminCredential $CloudAdminCredential -AzureSubscriptionId $SubscriptionId -JeaComputername "Azs-ERC01" -ResourceGroupName "ContosoStackRegistrations" -RegistrationName "Registration01"
+
+.Notes
+
+If you would like to un-Register with you azure by turning off marketplace syndication and usage reporting you can run this script again with both enableSyndication
+and reportUsage set to false. This will unconfigure usage bridge so that syndication isn't possible and usage data is not reported.
+
+If you would like to use a different subscription for registration you must remove the activation resource from Azure and then re-run this script with a new subscription Id passed in.
```
Comment (sbusayarat) [inline Registration/RegisterWithAzure.ps1]: `you must remove the activation resource from Azure` <= that step is not that straight forward since the user may not remember what Azure resource group name and registration name they used during the registration.

I talked to Yoav about this and we think we should add a proper Unregister/Deactivate script that perform all the steps properly.    That is out of scope of this PR.

**Comment by BradleyBartlett** [comment]
@sarathys  @sbusayarat 

Thanks for the feedback. I will submit a new review with a few more updates and fixes.
